### PR TITLE
Create camera using all indexes if auto detection fails

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/<ticket-number>) |

#### Main changes
-

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: packages/camera/pitop/camera/core/cameras/usb_camera.py [hunk 1]
```diff
@@ -46,12 +51,19 @@ def create_camera_object(index, resolution=None):
...
 
-            if resolution is not None:
+            if cap is not None and resolution is not None:
                 cap.set(3, resolution[0])
                 cap.set(4, resolution[1])
             return cap
 
-        self._camera = create_camera_object(self.index, resolution)
+        self._camera = create_camera_object(index, resolution)
+        if self._camera is None and index is None:
```
Comment (angusjfw) [inline packages/camera/pitop/camera/core/cameras/usb_camera.py:60]:
```suggestion
        if self._camera is None and index is -1:
```
we only want to scan if the index was not specified by the user - which would mean it is now -1
